src/lib/Tab/LocationView/AuthorsTab.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Tab\LocationView;
+
+use Ibexa\AdminUi\Specification\UserExists;
+use Ibexa\AdminUi\UI\Dataset\DatasetFactory;
+use Ibexa\Contracts\AdminUi\Tab\AbstractEventDispatchingTab;
+use Ibexa\Contracts\AdminUi\Tab\OrderedTabInterface;
+use Ibexa\Contracts\Core\Repository\SectionService;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo;
+use Ibexa\Contracts\Core\Repository\Values\Content\VersionInfo;
+use JMS\TranslationBundle\Annotation\Desc;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Environment;
+
+class AuthorsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
+{
+    public const URI_FRAGMENT = 'ibexa-tab-location-view-authors';
+
+    protected UserService $userService;
```

Comment:
```suggestion
    private UserService $userService;
```

Reply:
General rule is to use `private` as default modifier until you have reason to make some property/method protected or public.